src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -12979,3 +12979,79 @@ def test_aks_update_agentpool_os_sku(self, resource_group, resource_group_locati
             "aks delete -g {resource_group} -n {name} --yes --no-wait",
             checks=[self.is_empty()],
         )
+
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest',
```

Comment:
As the case is in preview and only works in staging at the moment, do you agree to exclude the case from the default test matrix (`src/aks-preview/azcli_aks_live_test/configs/ext_matrix_default.json`)? Otherwise, once you add these two cases, they will definitely fail in the cli runner (run cases according to the test matrix once every day), resulting in noise alerts.

Reply:
Added to the `ext_matrix_default.json`. I'll update this once the feature is rolled out to production regions